### ISSUE TITLE
IntelliSense Typings install for Express

### ISSUE THREAD

**Issue by tylerlrhodes** [Microsoft/vscode-docs#646]
In the Node JS guide for setting up Visual Studio Code there are a few steps for installing the typings for Node and express.  It seems to work for node, but not for express, and fails for finding express-serve-static-core

The following link shows how to install it, and then the IntelliSense works again.

https://www.npmjs.com/package/@types/express-serve-static-core

**Comment by gregvanl** [comment]
Hi @tylerlrhodes, the existing instructions for using the Typings command line still worked for me and I was able to get the typings installed for express. At some point, we'll update all the documentation to use `npm install @types` but I wasn't able to reproduce your issue.

**Comment by tylerlrhodes** [comment]
Greg, okay ... thanks for the feedback.  It didn't work for me until I took these steps, probably something in my environment I guess.  I'm running on macOS Sierra for any future reference, I have Node 4.6 LT edition.  Strange, because the typings install worked for me, however when I went to use the Intellisense feature it didn't come up properly.  I also noticed this for the oracledb node module too, which worked once I used the @types method.

Anyway, thanks for responding.

Tyler
